app/Resources/MgiletNotificationBundle/views/notification_dropdown.html.twig

```diff
@@ -0,0 +1,58 @@
+{% if mgilet_unseen_notification_count() > 0 %}
+<li class="btn-warning active">
+    {% else %}
+<li>
+    {% endif %}
+    <a href="#!" id="notificationDropdown" data-toggle="collapse" data-target="#notification-menu"
+       aria-haspopup="true">
+        <span class="glyphicon glyphicon-bell"></span>
+        <span id="notificationCount">
+            {{ mgilet_unseen_notification_count() }}
+        </span>
+    </a>
+    <div id="notification-menu" class="notification-menu collapse" aria-labelledby="notificationDropdown">
+        <ul class="well no-padding list-unstyled">
+            <li style="padding: 10px;">
+                <b class="text-muted">{{ 'notifications'|trans({},'MgiletNotificationBundle') }}</b>
+                <a id="notification-MarkAllAsSeen" href="{{ path('notification_mark_all_as_seen') }}"
+                   class="no-margin btn btn-xs pull-right">{{ 'mark_all_as_seen'|trans({},'MgiletNotificationBundle') }}
+                </a>
+            </li>
+            <hr class="no-margin">
```

Comment:
You should not put `hr` element inside the `ul` tag. There should be only `li`s.